modules/oe_whitelabel_modal_page/oe_whitelabel_modal_page.info.yml

```
@@ -0,0 +1,7 @@
+name: OpenEuropa Whitelabel Modal
+type: module
+description: Modal component.
```

What about
```suggestion
description: Integrates the Modal Page module with OpenEuropa Whitelabel theme.
```
(not sure if there is a convention for these descriptions)

---

Done in [9eafc3f](https://github.com/openeuropa/oe_whitelabel/pull/390/commits/9eafc3f353781dd2efd6f2f3ea79308e3732950b)